ietf/doc/views_draft.py

```diff
@@ -1250,6 +1209,46 @@ def edit_doc_extresources(request, name):
     title = "Additional document resources"
     return render(request, 'doc/edit_field.html',dict(doc=doc, form=form, title=title, info=info) )
 
+class ConsensusForm(forms.Form):
+    consensus = forms.BooleanField(required=True, label="When published as an RFC, should the consensus boilerplate be included?")
```

Comment:
With `required=True`, this field _must_ be checked or validation will fail. I don't think that's what was intended?

Reply:
Gross. I had been counting on tests - points out we need a test that exercises setting the state both ways. Howeverr, walking through the experience manually says I need to rework this _again_. I'll bring it back when refactored.